settings_file: "settings_sobec.yaml"
left_foot_name: "left_sole_link"
right_foot_name: "right_sole_link"
urdf: "/opt/openrobots/share/example-robot-data/robots/talos_data/robots/talos_reduced.urdf"
srdf: "/opt/openrobots/share/example-robot-data/robots/talos_data/srdf/talos.srdf"
controlled_joints:
  [
    root_joint,
    leg_left_1_joint,
    leg_left_2_joint,
    leg_left_3_joint,
    leg_left_4_joint,
    leg_left_5_joint,
    leg_left_6_joint,
    leg_right_1_joint,
    leg_right_2_joint,
    leg_right_3_joint,
    leg_right_4_joint,
    leg_right_5_joint,
    leg_right_6_joint,
    torso_1_joint,
    torso_2_joint,
    arm_left_1_joint,
    arm_left_2_joint,
    arm_left_3_joint,
    arm_left_4_joint,
    arm_left_5_joint,
    arm_left_6_joint,
    arm_left_7_joint,
    arm_right_1_joint,
    arm_right_2_joint,
    arm_right_3_joint,
    arm_right_4_joint,
  ]

lowerPositionLimit: [
    # Base
    -1, -1, -1, -1, -1, -1, -1,
    # Left leg
    -0.35, -0.52, -2.10, 0.0, -1.31, -0.52,
    # Right leg
    -1.57, -0.52, -2.10, 0.0, -1.31, -0.52,
    # Torso
    -1.3, -0.1,
    # Left arm
    -1.57, -0.2, -2.44, -2.1, -2.53, -1.3, -0.6,
    # Right arm
    -0.4, -2.88, -2.44, -2.1
  ]
upperPositionLimit: [
    # Base
    1, 1, 1, 1, 1, 1, 1,
    # Left leg
    1.57, 0.52, 0.7, 2.62, 0.77, 0.52,
    # Right leg
    0.35, 0.52, 0.7, 2.62, 0.77, 0.52,
    # Torso
    1.3, 0.78,
    # Left arm
    0.52, 2.88, 2.44, 0, 2.53, 1.3, 0.6,
    # Right arm
    1.57, -0.2, 2.44, 0
  ]
tool_frame_pos: [0, -0.02, -0.0825]
